src/agent/tools/lights_tools.py

```
@@ -34,38 +42,105 @@ def create_light(
+                design_level_calculation_method=design_level_calculation_method,
+                lighting_level=lighting_level if lighting_level != 0.0 else None,
+                watts_per_floor_area=watts_per_floor_area if watts_per_floor_area != 0.0 else None,
+                watts_per_person=watts_per_person if watts_per_person != 0.0 else None,
+                fraction_radiant=fraction_radiant,
+                fraction_visible=fraction_visible,
+            ))
+            return _ok(
+                f"Lights '{name}' created successfully.",
+                idf.get("Lights", name).model_dump(),
+            )
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)